src/components/TaskList.vue

```diff
@@ -32,26 +30,36 @@ import CreateTask from '@/components/CreateTask.vue';
 
 export default {
   name: 'TaskList',
-  props: ['listId', 'listTitle', 'tasks'],
+  props: ['listId'],
   components: {
     Task,
     CreateTask,
   },
+  computed: {
+    listTitle() {
+      const listIndex = this.$store.state.todoLists
+        .findIndex((list) => list.id === this.listId);
+      return this.$store.state.todoLists[listIndex].title;
+    },
```

Comment:
Easier to pass **title** property from parent component (like listId) than get it from store